### PR TITLE
write data model to JSON instead of TSV

### PR DESCRIPTION
**Pull request by smgogarten**

### CODE REVIEW

File: GREGoR_data_model.json [hunk 1]
```diff
@@ -0,0 +1,613 @@
+{
+  "name": "PRIMED Genotype Data Model",
```
Comment (bheavner) [inline GREGoR_data_model.json]: Should be GREGoR Consortium Data model

File: GREGoR_data_model.json [hunk 2]
```diff
@@ -0,0 +1,613 @@
...
+  "description": "Data model for genotype data in the PRIMED consortium",
```
Comment (bheavner) [inline GREGoR_data_model.json]: Should be GREGoR, not PRIMED

File: sheets_to_JSON.R [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+# 2) the list of data tables corresponding to the first argument
+source("sheets_to_list.R")
+tab_list <- sheets_to_list(apply(meta, 1, as.list), tables)
+rm(list = c("meta", "tables", "sheets_to_list"))
+
+
+# initialize leading text
+master <- list(
+    # Overall File Details
+    name = "PRIMED Genotype Data Model",
```
Comment (bheavner) [inline sheets_to_JSON.R]: perhaps name should be a constant set at the top, like the url (or updated for GREGoR)

File: sheets_to_JSON.R [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    description = "Data model for genotype data in the PRIMED consortium",
```
Comment (bheavner) [inline sheets_to_JSON.R]: Not true for GREGoR.

File: sheets_to_JSON.R [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    version = "0.1.2",
```
Comment (bheavner) [inline sheets_to_JSON.R]: Perhaps this should be read from the sheet/metadata tab?

Comment (bheavner) [reply sheets_to_JSON.R]: Though I'm not sure of that - depends what we're considering the authoratative source, I suppose. Another option would be to hard code it at the top, like URL.

**Review by bheavner — COMMENTED** [review]
A few fairly minor comments

File: sheets_to_JSON.R (hunk 3 above)
Comment (smgogarten) [reply sheets_to_JSON.R]: good idea, done

**Pull request merged**